### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by DankaKM**
Update the game over message so people know how to play again

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (OliverKovalcik) [inline index.html]: To co za CODE  ? ??

**Comment by OliverKovalcik** [comment]
Komentar

Komentar komentara

File: index.html (hunk 1 above)
Comment (OliverKovalcik) [inline index.html]: Cele zle

**Review by OliverKovalcik — COMMENTED** [review]
Koncim toto review

File: index.html (hunk 1 above)
Comment (DankaKM) [reply index.html]: to nemyslis vazne

File: .github/steps/-step.txt [hunk 2]
```diff
@@ -1 +1 @@
-0
+5
```
Comment (DankaKM) [inline .github/steps/-step.txt]: gfds

**Review by DankaKM — COMMENTED** [review]
dfgsdfgsdf

File: index.html (hunk 1 above)
Comment (DankaKM) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again hun 🧑‍💻 🤖!</h2>
```
pp

Comment (DankaKM) [reply index.html]: fgfgdfgjladf

**Pull request merged**